test/python/qasm3/test_export.py

```diff
@@ -494,7 +542,7 @@ def test_unbound_circuit(self):
         )
         self.assertEqual(Exporter().dumps(qc), expected_qasm)
 
-    def test_unknown_parameterized_gate_called_multiple_times(self):
+    def test_standard_parameterized_gate_called_multiple_times(self):
```

Comment:
I don't see a test for the case that first instance of the gate is **not** generic.
```python
        qc.rzx(0.5, 0, 1)
        qc.rzx(x, 0, 1)
        qc.rzx(y, 0, 1)
```
With this PR this is exported correctly!

Reply:
Thanks - cherry-picked as 2cbf15ff4d0ae35e845e9db9a66070597b14b679.